spec/lib/presenters/taxonomy_navigation_spec.rb

```diff
@@ -1,10 +1,11 @@
 require 'spec_helper'
 require 'webmock/rspec'
 require 'gds_api/test_helpers/rummager'
+require 'govuk_publishing_components/presenters/taxonomy_navigation'
```

Comment:
This shouldn't be necessary, the requiring should happen in `govuk_publishing_components.rb`.